Source/WTF/wtf/FixedVector.h

```diff
@@ -26,12 +26,15 @@
 #pragma once
 
 #include <wtf/EmbeddedFixedVector.h>
+#include <wtf/MallocCommon.h>
 
 namespace WTF {
 
 template<typename T, typename Malloc>
 class FixedVector {
+    WTF_MAKE_CONFIGURABLE_ALLOCATED(Malloc);
```

Comment:
It looks like WTF_MAKE_CONFIGURABLE_ALLOCATED(Malloc) is unnecessary here. FixedVector<T, Malloc> itself is never dynamically allocated, and Malloc is only used for managing the underlying storage, not the FixedVector instance. Should we remove this to avoid unnecessary complexity?

Reply:
I'm not convinced it won't be in the future, though -- SegmentedVector, for example, is quite similar but is dynamically allocated, and if someone decides to do the same for FixedVector I don't think we can assume that they'll remember to make it arena-allocated.